tools/test.py

```diff
@@ -44,11 +44,14 @@
 import errno
 import copy
 import ast
+import resource
 
 from os.path import join, dirname, abspath, basename, isdir, exists
 from datetime import datetime
 from Queue import Queue, Empty
 
+resource.setrlimit(resource.RLIMIT_CORE, (0,0))
```

Comment:
I'm thinking specialize `SimpleTestConfiguration` and do it there.
Maybe we'll want a test that examines the core-dump and deletes it?
Also there was an issue where we wanted the core for post-mortem...